base/operators.jl

```diff
@@ -189,7 +189,7 @@ copy(x::Union{Symbol,Number,AbstractString,Function,Tuple,LambdaStaticData,
               TopNode,QuoteNode,DataType,Union}) = x
 
 # function pipelining
-|>(x, f::Callable) = f(x)
+|>(x, f) = f(x)
```

Comment:
I don't think so, since Cmd piping will still invoke the more specific ``|>(src::Base.AbstractCmd, dest::Base.AbstractCmd)``.